src/formats/__tests__/frontmatter.spec.js

```diff
@@ -192,7 +192,7 @@ describe('Frontmatter', () => {
     );
   });
 
-  it('should stringify YAML with --- delimiters when it is explicitly set as the format without a custom delimiter', 
```

Comment:
If this file doesn't have any actual code changes, can we just clean up the diff?